element_data_loader/Suite2p_trigger_individual/Suite2p_trigger.py

```diff
@@ -0,0 +1,111 @@
+from suite2p import run_s2p, default_ops
+from suite2p.detection.anatomical import roi_detect
+from suite2p.extraction import dcnv
+import numpy as np
+import os
+
+
+def motion_correction_suite2p(ops, db):
+    """Performs registration using Suite2p package, 
+    Make sure the 'do_registration' key is set to 1 to perform registration
+
+    Args:
+        ops (dict): ops dictionary can be obtained by using default_ops() function. 
+                    It contains all options and default values used to perform 
+                    preprocessing.
+
+    Returns:
+        dict : Returns a dictionary with x and y shifts
+    """
+    # Rigid motion correction
+    if not ops['nonrigid']:
+        # The flags are hard coded and set to run each step of Suite2p (as this
+        # is meant to be fixed. If this is updated by the user, we can generate 
+        # an error warning to mention that this function is only meant to be
+        # used for registration.
+
+        ops.update(do_registration = 1, 
+                    roi_detect = False,
+                    spikedetect = False,
+                    delete_bin = True)
+
+        reg_ops = run_s2p(ops,db)
+        desired_keys = ['xoff', 'yoff', 'do_registration', 'two_step_registration',
+                        'roidetect', 'spikedetect', 'delete_bin']
+        reg_ops = {key: reg_ops[key] for key in desired_keys}
+
+        return reg_ops
+
+    else:
+        
+        ops.update(nonrigid = True, 
```

Comment:
Based on the if...else statement isn't the `nonrigid` key already declared properly?